### ISSUE TITLE
[Albion Online] CSD move behaves very badly

### ISSUE THREAD

**Issue by damianatorrpm** [WayfireWM/wayfire#953]
Works correctly on GNOME.

Albion implements it's own CSD (which "works"). Dragging it by the CSD to move the window works,
but it does "back-snapping".  It moves than quickly reverts it's position and moves back to the new position causing incredible flickers.

```
[tux@localhost ~]$ flatpak search albion
Name                   Description                                Application ID                         Version            Branch         Remotes
Albion Online          MMORPG open medieval fantasy game          com.albiononline.AlbionOnline          1.0.34.184         stable         flathub

```

**Comment by ammen99** [comment]
sounds pretty much a duplicate of #952, or is there something different here?

**Comment by damianatorrpm** [comment]
yes, it's different this one can be moved but it behaves wrong, #952 cant be moved at all.

**Comment by ammen99** [comment]
With recent changes, it cannot be moved anymore. I do not want to support clients which don't use the standard interfaces for window movement. As usual though, a plugin can override this decision. Duplicate of #952
